docs/ToDo.md

```diff
@@ -2,9 +2,12 @@
 
 
 ## High Priority TO-DO
-- add Django 
+- Figure out stats/losses/core code, currently a class in the views
+- Figure out how to create and close a connection for the dps threat lookup
 - DTO stuff, figure out 
 - Create Project folder, and app (project is riot-api, app is stats?)
+- Probably remove the Runner code
```

Comment:
Is the runner code provided by django?  I would have figured django gave you some kind of loop out of the box

Reply:
This first started out as just a single file script called Runner.py, then i moved the business logic code into another file and left Runner.py as the starting point. This past week I added Django, and then hooked everything up to flow from the Django manage.py script. There is still some old code floating around that needs to be nuked (aka Runner.py)